### ISSUE TITLE
Intellisense stops working often with no extensions installed.

### ISSUE THREAD

**Issue by waynebloss** [Microsoft/vscode#37731]
- VSCode Version: Code 1.17.2 (b813d12980308015bcd2b3a2f6efa5c810c33ba5, 2017-10-16T13:59:46.104Z)
- OS Version: Windows_NT x64 10.0.15063
- Extensions: none
---

Steps to Reproduce:

1. Use vscode for a while, editing .js and .jsx files in a create-react-app project.
2. At some point, try to import a module into a file and the only Intellisense that I get is `Loading...`.

I have to reload the window to make it work again. This has happened so often that I have bound a shortcut key to reload the window.

Following the instructions [here](https://github.com/Microsoft/vscode/issues/23665#issuecomment-291206258) I have captured the tsserver log and trace messages. For the captured session: 

1. I opened my project folder, opened a JSX file and typed `import {} from 'react-intl'`. 
2. Then I put my cursor in between the squigglies and pressed `ctrl+space`. I got proper Intellisense.
3. Then I removed that line and typed `import {} from 'cheerio'`. This time I got `Loading...` Intellisense.

NOTE: I replaced the name of my project, which is 8 letters long, with the word `MY_PROJECT` in the logs.

[vscode-tsserver-log-7frhTz.zip](https://github.com/Microsoft/vscode/files/1447550/vscode-tsserver-log-7frhTz.zip)

UPDATE: As I prepare to roll back to 1.16.1, I noticed that my `%TEMP%` folder is full of zero-byte files named something like `vscode-tscancellation-a6e8e635fc0b3155d7f3.sock31`...

UPDATE: My settings - 

```js
{
    // Updates
    "update.channel": "none",
    "extensions.autoUpdate": false,
    // Editor
    "editor.minimap.enabled": false,
    "editor.tabSize": 2,
    // Workbench
    "workbench.editor.enablePreview": false,
    "workbench.editor.enablePreviewFromQuickOpen": false,
    // Window
    "window.autoDetectHighContrast": false,
    "window.restoreWindows": "none",
    "window.zoomLevel": 0,
    // Files
    "files.eol": "\n",
    "files.hotExit": "off",
    // File Explorer
    "explorer.autoReveal": false,
    "explorer.openEditors.visible": 0,
    // Git
    "git.enabled": false,
    // JSHint
    // "jshint.enable": false,
    // "jshint.options": {
    //   "esversion": 5
    // },
    // Search
    "search.exclude": {
      "**/node_modules": true,
      "**/bower_components": true,
      "**/public": true,
      "**/.cache": true,
      "**/build": true
    },
    // Telemetry
    "telemetry.enableTelemetry": false,
    "telemetry.enableCrashReporter": false,
    "workbench.colorTheme": "Visual Studio Dark",
    // "java.errors.incompleteClasspath.severity": "ignore",
    // TypeScript
    "javascript.validate.enable": false,
    "typescript.validate.enable": false,
    "typescript.tsserver.log": "off",
    "typescript.tsserver.trace": "off"
    // "workbench.iconTheme": "vscode-icons"
  }
```

**Comment by waynebloss** [comment]
At this time I have uninstalled VSCode, deleted all of my `%TEMP%` files and reinstalled VSCode and I am still able to reproduce the issue within about a minute of opening my project.

**Comment by waynebloss** [comment]
After rolling back to 1.16.1, I can no longer reproduce the issue. If there is some change, I will note it here.

I noticed this problem as soon as I updated to 1.17, which is why I went back to 1.16. Also, I used to run the 32-bit version of 1.16 but switched to 64-bit for 1.17. However, when I rolled back - I installed 1.16.1-x64.

Each time I roll back, I delete all of my `%TEMP%` files *(or at least the ones that look to be related to vscode)* as well as the `%AppData%\Code` directory. When I upgrade, I don't do anything special.
